bin/rialto/runtime/src/lib.rs

```diff
@@ -414,7 +414,20 @@ parameter_types! {
 	// Note that once this is hit the pallet will essentially throttle incoming requests down to one
 	// call per block.
 	pub const MaxRequests: u32 = 50;
+}
 
+#[cfg(feature = "runtime-benchmarks")]
+parameter_types! {
+	// Number of headers to keep in benchmarks.
+	//
+	// In benchmarks we always populate with full number of `HeadersToKeep` to make sure that
+	// pruning is taken into account.
+	// This is lower than regular value, to speed up benchmarking setup.
+	pub const HeadersToKeep: u32 = 1024;
+}
```

Comment:
This code is optional, but keeping it reduces the time to run benchmarks ~30% (from 70minutes to 54minutes).

Reply:
Is the 70 mins for the full (messages + header) bench suite?